Lib/test/test_pickle.py

```diff
@@ -118,13 +118,9 @@ def test_in_band_buffers(self): # TODO: RUSTPYTHON, remove when this passes
     def test_nested_names(self): # TODO: RUSTPYTHON, remove when this passes
         super().test_nested_names() # TODO: RUSTPYTHON, remove when this passes
 
-    # TODO: RUSTPYTHON, AssertionError
-    @unittest.expectedFailure
```

Comment:
When you delete this, can you also delete the two lines that say `# TODO: RUSTPYTHON, remove when this passes`?